### PR TITLE
New amp-ad for aniview

### PR DESCRIPTION
**Pull request by itaynave**
new amp-ad type="aniview"

### CODE REVIEW

**Comment by itaynave** [comment]
resubmitted pull request to fix CLA issue

**Comment by itaynave** [comment]
any update on that?
Is anything missing?

**Comment by itaynave** [comment]
> Please also list `aniview` in `extensions/amp-ad/amp-ad.md` file. Otherwise LGTM

Done

File: ads/aniview.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/aniview.js]: Incorrect copyright year 😄

File: ads/aniview.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/aniview.md]: Incorrect copyright year 😄

**Comment by torch2424** [comment]
@itaynave Just wanted to check in on this. Did you still want to get this PR in? Thanks! 😄

**Comment by itaynave** [comment]
Yes please.
Thank you

File: ads/aniview.md (hunk 2 above)
Comment (itaynave) [reply ads/aniview.md]: Fixed year

**Comment by torch2424** [comment]
@itaynave Awesome! Thanks for making the changes, and for making this contribution! 😄 👍 🎉 

Merging...

**Pull request merged**